renderer.php

```diff
@@ -490,7 +490,7 @@ public function course_image_defaults() {
             $image = html_writer::empty_tag('img', array( 'src' => $imageurl, 'class' => 'course_image' ));
             return html_writer::div($image, 'image_wrap');
         } else {
-            // We need a CSS soloution apparently lets give it to em.
+            // We need a CSS solution apparently lets give it to 'em.
```

Comment:
```suggestion
            // We need a CSS solution apparently. Lets give it to 'em.
```

Or X gon' give it to ya.